docs/using-template-repos-for-assignments.md

```diff
@@ -0,0 +1,48 @@
+Importing starter code for assignments is a very useful feature and with GitHub recently shipping the template repositories, we can leverage them in Classroom to make importing starter code much easier and faster!
+
+### How to use template repositories for an assignment?
+
+#### Before you begin
+To use template repositories for assignments, you'll need the following:
+
+* Set-up a GitHub classroom
+* An exercise (a repository that you have access to, which contains documentation, starter code, tests, or anything else your students need to begin work on an assignment).
+
+#### Get started
+
+Once you log in to [GitHub Classroom](https://classroom.github.com/), select your classroom and then click on **New Assignment**. Template repositories work for both **Individual Assignments** and **Group Assignments**. So you can pick either of the two for your assignment. You can then provide details for the assignment. Once you've filled out the details, the **Optional** section has a search bar for finding your starter code repository. When you search for your starter code repository, you will now get two options:
+
+  * Import starter code using a template repository.
+  * Import starter code using source importer.
+
+If you choose the first option, we will use your starter code repository as a template for generating the assignment for the students.
+
+![Searches for repository, then selects the option of Importing starter code with template repositories](https://user-images.githubusercontent.com/3170078/62091261-ee69f900-b224-11e9-96ab-b4b34cce0001.gif)
+
+Note: For import via template repositories to work, your starter code repository _must_ be a template repository. If you own the repository or have permissions, you can do it manually from the settings page of the repository on GitHub. If you do not own the repository, you can make a fork of it and then make it a template. More details [here](https://help.github.com/en/articles/creating-a-template-repository). We are in the process of automating this for you, and will post an update on this page soon!
+
+
+### What are the advantages of using template repositories?
+
+* Importing starter code is faster, as it uses the new [template repositories API](https://developer.github.com/v3/repos/#create-repository-using-a-repository-template).
+* Template repositories are more reliable as we have noticed that 99% of our imports with template repositories succeeded, whereas only 95% of imports with source importer succeeded.
```

Comment:
I know I just suggested we say template repos succeed 99% of the time, but reading now, I feel like a teacher of a class of 100 is going to read this as "1 student is going to get stuck every single assignment". Why don't we remove the percentages and go with this instead:

```suggestion
* Template repositories are far more reliable than source imports.
```